### ISSUE TITLE
having Broadcast group 

### ISSUE THREAD

**Issue by soheilamiri** [microsoft/terminal#17011]
# create a feature for broadcasting to group of tabs or splited windows.

# Proposed technical implementation details (optional)

**Comment by zadjii-msft** [comment]
Do you mean something beyond [Broadcast input](https://learn.microsoft.com/en-us/windows/terminal/customize-settings/actions#broadcast-input)/?

* as first referenced in: #2634
* as implemented in #14393
* with additional follow ups in #16600

**Comment by soheilamiri** [comment]
> Do you mean something beyond [Broadcast input](https://learn.microsoft.com/en-us/windows/terminal/customize-settings/actions#broadcast-input)/?
> 
> * as first referenced in: [Support broadcast input? #2634](https://github.com/microsoft/terminal/issues/2634)
> * as implemented in [Add support for broadcasting to all panes in a tab #14393](https://github.com/microsoft/terminal/pull/14393)
> * with additional follow ups in [[Megathread] Broadcast input follow-ups #16600](https://github.com/microsoft/terminal/issues/16600)

yes, i mean some thing beyond normal [broadcast input ](https://learn.microsoft.com/en-us/windows/terminal/customize-settings/actions#broadcast-input) definition.
When broadcast mode is enabled, all input sent to the pane will be sent _to all panes in the same tab_. This is useful for sending the same input to multiple panes at once.
but i my request is adding a feature that enable me to grouping the panes, adding tag or name to group and input send to selected group or groups. panes can be on same tab or spread on multiple tab.

![Screenshot 2024-04-03 214956](https://github.com/microsoft/terminal/assets/86443621/9d60227e-d52d-495d-bcb4-36f8795fab40)
